src/main/java/Duke/ToDo.java

```diff
@@ -0,0 +1,76 @@
+package Duke;
+
+/**
+ * The ToDo class represents a to-do task for the Duke program.
+ * It extends the SingleTask class and provides additional methods specific to to-do tasks.
+ */
+public class ToDo extends SingleTask {
+    public ToDo(String s) {
+        super(s);
+    }
+
+    /**
+     * Marks the task as done and displays a message to the user.
+     */
+    public String mark() {
+        this.isDone = true;
+        return "Ok boy i mark for you already \n" +
+                "[" +this.getStatusIcon() +"] " + this.description;
```

Comment:
Just a small point, but I believe this is missing a whitespace between the `+` and the `this...`. This recurs in a few other places too.